frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.html

```diff
@@ -6,200 +6,172 @@
+            <div *ngIf="!language.getRawValue()" class="placeholder" style="z-index: 1">Native Language</div>
+            <label class="label" *ngIf="language.getRawValue()">Native Language</label>
+            <ng-select class="custom" dropdownPosition="bottom" appearance="underline" #languageSelect
+                       formControlName="language">
+                <ng-option class="select-option" *ngFor="let lang of languages" value="{{lang}}">{{lang}}</ng-option>
             </ng-select>
             <mat-icon class="expand-icon">expand_more</mat-icon>
         </div>
 
         <div class="form-input-group">
             <div *ngIf="!languageLevel.getRawValue()" class="placeholder">Level in English</div>
-            <ng-select
-                class="custom"
-                dropdownPosition="bottom"
-                appearance="underline"
-                #englishLevelSelect
-                formControlName="languageLevel">
-                <ng-option class="select-option" *ngFor="let level of languageLevelOptions" value="{{ level }}">
-                    {{ level }}
+            <label class="label" *ngIf="languageLevel.getRawValue()">Level in English</label>
+            <ng-select class="custom" dropdownPosition="bottom" appearance="underline" #languageLevelSelect
+                       formControlName="languageLevel">
+                <ng-option class="select-option" *ngFor="let level of languageLevelOptions" value="{{level}}">
+                    {{level}}
                 </ng-option>
             </ng-select>
             <mat-icon class="expand-icon">expand_more</mat-icon>
         </div>
     </div>
-    <div class="user-interests-section"></div>
-    <div class="user-socials-section">
-        <div class="form-input-group">
-            <label class="label" *ngIf="email.getRawValue()">Email</label>
-            <div *ngIf="!email.getRawValue()" class="placeholder">Email</div>
-            <input readonly class="input" type="text" id="email" formControlName="email" />
-        </div>
-
-        <div class="form-input-group">
-            <label class="label" *ngIf="instagram.getRawValue()">Instagram</label>
-            <div *ngIf="!instagram.getRawValue()" class="placeholder">Instagram</div>
-            <input class="input" type="text" id="instagram" formControlName="instagram" />
-        </div>
-
-        <div class="form-input-group">
-            <label class="label" *ngIf="facebook.getRawValue()">Facebook</label>
-            <div *ngIf="!facebook.getRawValue()" class="placeholder">Facebook</div>
-            <input class="input" type="text" id="facebook" formControlName="facebook" />
+    <div class="user-interests-section">
+        <div *ngFor="let item of getStringTags(); let i = index" class="user-interest-content">
+            <input type="checkbox" id="{{i}}" [checked]=includesTags(item)
+                   class="item-checkbox"
+                   (change)="selectInterest($event)" formControlName="tags"/>
```

Comment:
You can pass your parameters to events, like index for example
selectInterest($event, i)

Reply:
fixed